quickstart/docker-compose.yml

```diff
@@ -135,7 +135,7 @@ services:
     - ALLOW_EMPTY_PASSWORD=yes
     - REDIS_DISABLE_COMMANDS=FLUSHDB,FLUSHALL
   mongodb:
-    image: mongo:${MONGODB_VERSION:-6.0.5}
+    image: mongo:${MONGODB_VERSION:-7.0.2}
```

Comment:
this should not be updated manually. It happens in CI.

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/3301/commits/ca426c4de10bd79733d73deb51d87b5bd541469f